admob/src/ios/interstitial_ad_internal_ios.mm

```diff
@@ -32,29 +32,20 @@
 
 InterstitialAdInternalIOS::~InterstitialAdInternalIOS() {
   firebase::MutexLock lock(mutex_);
-  // Clean up any resources created in InterstitialAdInternalIOS.
```

Comment:
This code was waiting to run on the main thread but all it was doing was setting values to null.  While all AdMob operations should be conducted on the main thread, we're not actually performing any AdMob operations here, so let's simplify this code and just assign the null values here.